src/color-slider/color-slider.js

```diff
@@ -114,6 +117,15 @@ export default class ColorSlider extends HTMLElement {
 				this._el.spinner.style.setProperty("--value-length", valueStr.length);
 			}
 		}
+
+		if (name === "color" || name === "defaultColor") {
+			let color = this.color;
+
+			if (color) {
+				let displayedColor = color.display();
+				this.style.setProperty("--color", displayedColor);
+			}
+		}
```

Comment:
This is a duplicate of lines 103–110 above. Do we need this logic twice, or should we remove either (I don't see why the order matters)?